light_curves/requirements_light_curve_generator.txt

```
@@ -14,6 +14,9 @@ astroquery>=0.4.8.dev0
 acstools
 lightkurve
 alerce
+lsdb
+#s3fs
+#nested_pandas
```

I'm sorry for nitpicking, but these need to be removed and not commented out. Rule of thumb, only leave commented out lines when they serve a clear purpose or a message to the user. Leaving in stuff that are not needed any more doesn't fall in that category.
```suggestion
lsdb
dask[distributed]
```